decidim-core/spec/lib/map/autocomplete_spec.rb

```diff
@@ -57,8 +57,32 @@ module Map
           { url: "#{Decidim::Dev::Test::MapServer.host}/photon_api" }.to_json
         )
         expect(form_markup).to include(%(
-          <input autocomplete="off" data-decidim-geocoding="#{config}" type="text" name="test[address]" id="test_address" />
-        ).strip)
+      <input autocomplete="off" data-decidim-geocoding="#{config}" type="text" name="test[address]" id="test_address" />
+    ).strip)
+      end
+
+      context "when show_my_location_button? returns true" do
```

Comment:
```suggestion
      context "when show_my_location_button? returns an array" do
```
We should have also a test case for the `:all` case